app/assets/stylesheets/govuk_publishing_components/components/_button.scss

```diff
@@ -25,15 +25,33 @@ $gem-quiet-button-hover-colour: darken($gem-quiet-button-colour, 5%);
 .gem-c-button--inline {
   display: block;
   margin-bottom: govuk-spacing(1);
+  vertical-align: baseline;
```

Comment:
This is causing this layout issue at certain sizes (as discussed).

![Screen Shot 2019-06-12 at 13 02 15](https://user-images.githubusercontent.com/861310/59349763-5b2b4180-8d12-11e9-9a21-4c711094a04c.png)